ssnm/templates/map_page.html

```diff
@@ -1,31 +1,33 @@
 {% extends "base.html" %}
 {% load bootstrap %}
-{% load static from staticfiles %}
+{% load compress %}
```

Comment:
good catch. i was going to use it for the jquery files, but then realized they were not in use.